packages/contracts-bedrock/test/BlockOracle.t.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Test } from "forge-std/Test.sol";
+import { BlockOracle } from "src/dispute/BlockOracle.sol";
+import "src/libraries/DisputeTypes.sol";
+import "src/libraries/DisputeErrors.sol";
```

Comment:
nice use of global import paths

Reply:
fwiw, created a ticket to track: https://linear.app/optimism/issue/CLI-4324/ctb-use-global-import-paths-and-document-in-style-guide